<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newDownloadDialog</class>
 <widget class="QDialog" name="newDownloadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>551</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 240, 245);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>18</y>
     <width>23</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 240, 245);
border-image: url(:/new/mainwindow/pictures/system_icon.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>17</y>
     <width>200</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 240, 245);</string>
   </property>
   <property name="text">
    <string>确认下载</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>536</x>
     <y>12</y>
     <width>13</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 240, 245);
border-image: url(:/new/mainwindow/pictures/exit_button.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>44</y>
     <width>551</width>
     <height>462</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>75</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/mainwindow/pictures/system_question.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>82</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>是否确认下载以下列表中的文件？</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonconfirm">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>460</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtoncancel">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>460</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton_close</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>listWidget</zorder>
  <zorder>pushButtonconfirm</zorder>
  <zorder>pushButtoncancel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
